src/app/js/admin/Appbar/ClearDialog.js

```diff
@@ -38,6 +38,9 @@ const ClearDialogComponent = props => {
 
     useEffect(() => {
         if (succeeded) {
+            window.location.replace(
```

Comment:
You sure we want that? It'll trigger a full page reload, not a react-router navigation